modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplAlternateURLTest.java

```diff
@@ -191,6 +210,33 @@ private ThemeDisplay _getThemeDisplay(Group group, String portalURL)
 		return themeDisplay;
 	}
 
+	private ThemeDisplay _getThemeDisplayWithVirtualHosts(
```

Comment:
Can you rename this to a plain `_getThemeDisplay`?

Reply:
_getThemedisplay already exists in the context and the new method uses the same amount of parameters. That's why I created a new one for the virtualhost related test. I didn't want to change the current working of the previous tests.

Reply:
Nevermind them. Looks reasonable.